.travis.yml

```diff
@@ -1,19 +1,22 @@
 language: node_js
+dist: bionic
 
-node_js: 12
+node_js: 16
```

Comment:
Because you're adding `.nvmrc`, you can omit `node_js` in which case travis will use .nvmrc. This might be nice to stick to the same specific version and makes sure we don't get out of sync (e.g., updating nvmrc but not .travis.yml)

https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions-using-nvmrc

Reply:
That's good, I did not even think about the possibility of travis using nvmrc.
Thanks!